mRemoteV1/Config/Serializers/ConnectionSerializers/MsSql/DataTableSerializer.cs

```diff
@@ -108,6 +108,7 @@ private void CreateSchema(DataTable dataTable)
             dataTable.Columns.Add("PostExtApp", typeof(string));
             dataTable.Columns.Add("MacAddress", typeof(string));
             dataTable.Columns.Add("UserField", typeof(string));
+            dataTable.Columns.Add("Favorite", typeof(bool));
```

Comment:
Should connection favorites be shared among all users or specific to each user? If we both share a SQL DB connection list and I set connection "app-001.example.com" as a favorite, should you also see that as a favorite? I could see reasons for wanting this to be a personal or shared setting. 

If you think that sharing the favorites is important, you can leave this as it is.
If you think it should be a personal setting, revert the current changes to the database/schema files and instead update the following:
- `LocalConnectionPropertiesModel`
- `SqlConnectionsLoader.ApplyLocalConnectionProperties(ContainerInfo rootNode)`
- `SqlConnectionsSaver.PropertyIsLocalOnly(string property)`
- `SqlConnectionsSaver.UpdateLocalConnectionProperties(ContainerInfo rootNode)`

(Being forced to type out the steps to add a single property makes me realize we need to refactor the local connection properties code. It's ridiculous that we need to touch 4 completely separate places for 1 change.)

Reply:
I'll turn the favorite property into a local property - makes much more sense for every user to be able to set favorites for himself